onnx_tf/handlers/backend/gather.py

```diff
@@ -3,12 +3,25 @@
 from onnx_tf.handlers.backend_handler import BackendHandler
 from onnx_tf.handlers.handler import onnx_op
 from onnx_tf.handlers.handler import tf_func
+from .gather_and_scatter_mixin import GatherAndScatterMixin
 
 
 @onnx_op("Gather")
 @tf_func(tf.gather)
-class Gather(BackendHandler):
+class Gather(GatherAndScatterMixin, BackendHandler):
 
   @classmethod
   def version_1(cls, node, **kwargs):
     return [cls.make_tensor_from_onnx_node(node, **kwargs)]
+
+  @classmethod
+  def version_11(cls, node, **kwargs):
+    x = kwargs["tensor_dict"][node.inputs[0]]
+    indices = kwargs["tensor_dict"][node.inputs[1]]
+    axis = node.attrs.get("axis", 0)
+    result = cls.chk_idx_out_of_bounds_along_axis(x, axis, indices)
+    msg = 'Gather indices are out of bounds, please double check the indices and retry.'
+    with tf.control_dependencies([tf.compat.v1.assert_equal(result, True, message=msg)]):
+      indices = cls.process_neg_idx_along_axis(x, axis, indices)
+      kwargs["tensor_dict"][node.inputs[1]] = indices
+      return [cls.make_tensor_from_onnx_node(node, **kwargs)]
```

Comment:
@winston-zillow all the updated inputs and attrs need to send to cls.make_tensor_from_onnx_node.
You may use the Split https://github.com/onnx/onnx-tensorflow/blob/master/onnx_tf/handlers/backend/split.py handler as your reference here.